File: src/main/java/ru/reosfire/wmlib/guis/Inventory/Components/CommandButton.java
package ru.reosfire.wmlib.guis.Inventory.Components;

import org.bukkit.event.inventory.InventoryClickEvent;
import ru.reosfire.wmlib.guis.Inventory.Gui;
import ru.reosfire.wmlib.yaml.common.gui.CommandButtonConfig;

public class CommandButton extends Button
{
    private final CommandButtonConfig _config;
    public CommandButton(CommandButtonConfig config, Gui gui)
    {
        super(config, gui);
        _config = config;
    }

    @Override
    protected void OnClick(InventoryClickEvent event)
    {
        for (String command : _config.Command)
        {
            gui.Player.performCommand(command);
        }
    }
}

File: src/main/java/ru/reosfire/wmlib/guis/inventory/components/CommandButton.kt
package ru.reosfire.wmlib.guis.inventory.components

import org.bukkit.event.inventory.InventoryClickEvent
import ru.reosfire.wmlib.yaml.common.gui.CommandButtonConfig
import ru.reosfire.wmlib.guis.inventory.Gui

class CommandButton(private val _config: CommandButtonConfig, gui: Gui) : Button(_config, gui) {
    override fun onClick(event: InventoryClickEvent) {
        for (command in _config.commands) {
            gui.Player.performCommand(command)
        }
    }
}